SU2_DEF/src/SU2_DEF.cpp

```diff
@@ -263,11 +263,34 @@ int main(int argc, char *argv[]) {
   /*--- Surface grid deformation using design variables ---*/
   
   for (iZone = 0; iZone < nZone; iZone++){
+
+    /*--- Initialize total deformation of design variables to zero ---*/
+    su2double **TotalDeformation;
+    unsigned short iDV, iDV_Value;
+    TotalDeformation = new su2double*[config_container[iZone]->GetnDV()];
+
+    for (iDV = 0; iDV  < config_container[iZone]->GetnDV(); iDV++){
+      TotalDeformation[iDV] = new su2double[config_container[iZone]->GetnDV_Value(iDV)];
+      for (iDV_Value = 0; iDV_Value < config_container[iZone]->GetnDV_Value(iDV); iDV_Value++){
+        TotalDeformation[iDV][iDV_Value] = 0.0;
```

Comment:
You can use the following syntax to initialize to zero:
`TotalDeformation[iDV] = new su2double[config_container[iZone]->GetnDV_Value(iDV)] ();`
(note the () at the end).